Source/Core/check.js

```diff
@@ -0,0 +1,111 @@
+/*global define*/
+define([
+    './defined',
+    './DeveloperError',
+    './isArray'
+], function(defined,
+            DeveloperError,
+            isArray) {
+    'use strict';
+
+    var exports = {
```

Comment:
Also follow the Cesium approach for declaring a class.  For example look at `Cartesian3` and consider just the static functions.  One day this is likely to change to be more Node-like, but let's stay consistent until then.